Modules/Core/Common/include/itkConstShapedNeighborhoodIterator.h

```diff
@@ -374,8 +374,15 @@ class ITK_TEMPLATE_EXPORT ConstShapedNeighborhoodIterator : private Neighborhood
   /** Add non-zero neighborhood offsets to the active list. The
    * radius of the neighborhood must match the radius of the shaped
    * iterator */
+  template <typename NeighborPixelType>
```

Comment:
Nitpick: ITK's naming convention has always been to start of typename template argument names by `T`. So I would suggest something like `TNeighborPixel`.  (Unless you would like to abandon this old naming convention, of course.)